app/src/containers/OrderReviewPage.tsx

```diff
@@ -94,8 +100,11 @@ class OrderReviewPage extends React.Component<RouteComponentProps, OrderReviewPa
   }
 
   fetchOrderData() {
+    const { match } = this.props;
+    const cartCode = match.params.cart;
+    const cart = cartCode || 'default';
     login().then(() => {
-      cortexFetch(`/?zoom=${zoomArray.sort().join()}`,
+      cortexFetch(`/carts/${Config.cortexApi.scope}/${cart}?zoom=${zoomArray.sort().join()}`,
```

Comment:
Same comment as above here. We could also switch out the entire url to fall back to `defaultcart` as another option.